README.md

```diff
@@ -109,12 +109,43 @@ The template for the `blurb add` message looks like this:
 Here's how you interact with the file:
 
 * Add the GitHub issue number for this commit to the
-  end of the `.. gh-issue:` line.
+  end of the `.. gh-issue:` line. The issue can also
+  be specified via the ``-i/--issue`` option:
+
+  ```shell
+  blurb add -i 109198
+  # or equivalently
+  blurb add -i https://github.com/python/cpython/issues/109198
+  ```
 
 * Uncomment the line with the relevant `Misc/NEWS` section for this entry.
   For example, if this should go in the `Library` section, uncomment
   the line reading `#.. section: Library`.  To uncomment, just delete
-  the `#` at the front of the line.
+  the `#` at the front of the line.  Alternatively, the section can
+  be specified via the ``-s/--section`` option:
+
+  ```shell
+  blurb add -s "Library"
+  # or equivalently
+  blurb add -s 3
+  ```
+
+  The section can be referred to from its name (case insensitive) or its ID
+  defined according to the following table:
+
+  | ID | Section           |
+  |----|-------------------|
+  |  1 | Security          |
+  |  2 | Core and Builtins |
+  |  3 | Library           |
+  |  4 | Documentation     |
+  |  5 | Tests             |
+  |  6 | Build             |
+  |  7 | Windows           |
+  |  8 | macOS             |
+  |  9 | IDLE              |
+  | 10 | Tools/Demos       |
+  | 11 | C API             |
```

Comment:
I'm in two minds about adding ID numbers. What do others think?

Reply:
Well.. I personally always contribute to either section 2 3 or 4 and those sections are typically the ones with the longest names... The tool still offers the possibility for partial matching, namely `-s core` or `-s lib`, (technically `-s l` is fine as well because there's only one section that is matched so... it's pretty good IMO).

I can remove the functionality though if you think it's too much!

Reply:
I don't think the section numbers are a good idea.  Partial matching is good, as long as it raises an exception if there's more than one match (which I believe it does on line 928).

Reply:
Yes, multiple matching with partial matching raises an exception. Any reason why numbers aren't good actually? I may be biased, but I'm faster typing a number than the first letters I'd usually use (-s3 rather than -sl) [I'll remove them tomorrow though, I think it would still be an overkill for most users and it's probably better to *see* what section you are indeed using]